spec/objects/group_spec.rb

```diff
@@ -44,7 +44,7 @@
     it 'builds images url' do
       expect(subject.urls(size: '200x200').first).
         to eq(
-          "http://www.ucarecdn.com/#{ subject.uuid }/nth/0/-/resize/200x200/"
```

Comment:
Thanks, I'll look into changing this. I reasoned that since it was static already that it just needed an update.